### PR TITLE
Fsi updates 111820c

### PR DESCRIPTION
**Pull request by jbranson000**
https://github.com/adobe/xdm/issues/1017

### CODE REVIEW

**Comment by cmathis** [comment]
Put all your mixins in the correct folder. Note you have some sitting outside the experience-event folder.
A components/mixins/experience-event/experienceevent-claim-process.example.1.json (1)
A components/mixins/experience-event/experienceevent-claim-process.schema.json (66)
A components/mixins/experience-event/experienceevent-contact-request-details.example.1.json (1)
A components/mixins/experience-event/experienceevent-contact-request-details.schema.json (103)
A components/mixins/experience-event/experienceevent-credit-limit-increase-details.example.1.json (1)
A components/mixins/experience-event/experienceevent-credit-limit-increase-details.schema.json (36)
A components/mixins/experience-event/experienceevent-deposit-details.example.1.json (1)
A components/mixins/experience-event/experienceevent-deposit-details.schema.json (41)
A components/mixins/experience-event/experienceevent-dispute-steps.example.1.json (1)
A components/mixins/experience-event/experienceevent-dispute-steps.schema.json (36)
A components/mixins/experienceevent-balance-transfer-details.schema.json (36)
A components/mixins/experienceevent-bill-pay-steps.schema.json (36)
A components/mixins/experienceevent-claim-details.schema.json (63)
A components/mixins/experienceevent-contact-request-details.schema.json (51)
A components/mixins/experienceevent-deposit-details.schema.json (36)
A components/mixins/experienceevent-dispute-steps.schema.json (31)
A components/mixins/experienceevent-financial-account-creation-steps.schema.json (31)
A components/mixins/experienceevent-loyalty-details.schema.json (61)
A components/mixins/experienceevent-paperless-enrollment-steps.schema.json (31)
A components/mixins/experienceevent-quiz-steps.schema.json (31)
A components/mixins/experienceevent-quote-details.schema.json (46)
A components/mixins/experienceevent-referral-steps.schema.json (31)
A components/mixins/experienceevent-refinance-details.schema.json (31)
A components/mixins/experienceevent-social-network-usage-details.schema.json (56)
A components/mixins/experienceevent-trade-details.schema.json (31)
A components/mixins/experienceevent-user-login-details.schema.json (101)
A components/mixins/experienceevent-watchlist-steps.schema.json (31)

File: components/datatypes/financial-account.schema.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  "$id": "https://ns.adobe.com/xdm/datatypes/financial-account",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Financial Account",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:industry": ["financial_services"],
+  "description": "Financial account details, such as owners, balances, and types.",
+  "definitions": {
+    "financialAccount": {
+      "type": "object",
```
Comment (mjasrotia) [inline components/datatypes/financial-account.schema.json:17]: definition do not need to have a type

File: components/datatypes/policy-details.schema.json [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  "$id": "https://ns.adobe.com/xdm/datatypes/policy-details",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Policy Details",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:industry": ["financial_services"],
+  "description": "Details for the insurance policy such as type, start date, and ID.",
+  "definitions": {
+    "policy-details": {
+      "type": "object",
```
Comment (mjasrotia) [inline components/datatypes/policy-details.schema.json:17]: Definitions do not need to have a type. Please fix this across all the files.

File: components/datatypes/policy-details.schema.json [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+          "type": "string",
+          "description": "Insurance policy name."
+        },
+        "xdm:policyStartDate": {
+          "title": "Policy Start Date",
+          "type": "string",
+          "format": "date-time",
+          "description": "Date which the insurance policy coverage started."
+        },
+        "xdm:policyState": {
```
Comment (mjasrotia) [inline components/datatypes/policy-details.schema.json:41]: Do you want this to be open text or would like to use a pattern the way we use in the "geo.schema". 
"pattern": "([A-Z]{2}-[A-Z0-9]{1,3}|)" ,
"examples": ["US-CA", "DE-BB", "JP-13"],

File: components/datatypes/policy-details.schema.json [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+          "title": "Policy State",
+          "type": "string",
+          "description": "State in which the insurance policy is issued."
+        },
+        "xdm:policyType": {
```
Comment (mjasrotia) [inline components/datatypes/policy-details.schema.json:46]: type missing

File: components/datatypes/policy-details.schema.json (hunk 3 above)
Comment (jbranson000) [reply components/datatypes/policy-details.schema.json:40]: We can leave it as open text for now.

File: components/datatypes/policy-details.schema.json [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+          "title": "Policy Type",
```
Comment (mjasrotia) [inline components/datatypes/policy-details.schema.json:46]: type still missing

File: components/datatypes/tool-usage.schema.json [hunk 6]
```diff
@@ -0,0 +1,94 @@
+{
```
Comment (mjasrotia) [inline components/datatypes/tool-usage.schema.json:1]: please remove this from the PR if already pushed to INT

File: components/datatypes/transaction.schema.json [hunk 7]
```diff
@@ -0,0 +1,54 @@
+{
```
Comment (mjasrotia) [inline components/datatypes/transaction.schema.json:1]: please remove if already pushed to INT from a prev PR

File: components/mixins/experience-event/experienceevent-claim-process.schema.json [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+  "meta:industry": [
+    "financial_services",
+    "health_and_life_sciences",
+    "high_tech"
+  ],
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "description": "Claim details such as claim policy ID and claim creation steps.",
+  "definitions": {
+    "claim-process": {
+      "type": "object",
```
Comment (mjasrotia) [inline components/mixins/experience-event/experienceevent-claim-process.schema.json:23]: not required as its a definition

File: components/mixins/experience-event/experienceevent-claim-process.schema.json [hunk 9]
```diff
@@ -0,0 +1,68 @@
...
+      "properties": {
```
Comment (mjasrotia) [inline components/mixins/experience-event/experienceevent-claim-process.schema.json:24]: type and properties not required as its a definition

File: components/mixins/experience-event/experienceevent-claim-process.schema.json [hunk 10]
```diff
@@ -0,0 +1,68 @@
...
+        "xdm:disputes": {
+          "type": "object",
+          "properties": {
+            "xdm:claimProcess": {
+              "type": "object",
```
Comment (mjasrotia) [inline components/mixins/experience-event/experienceevent-claim-process.schema.json:29]: properties missing

File: components/mixins/experience-event/experienceevent-claim-process.schema.json [hunk 11]
```diff
@@ -0,0 +1,68 @@
...
+                "type": "string",
+                "description": "ID of the claim being submitted."
+              },
+              "xdm:claimStartDate": {
+                "title": "Claim Start Date",
+                "type": "string",
+                "format": "date-time",
+                "description": "Date the claim was successfully submitted."
+              },
+              "xdm:claimStatus": {
```
Comment (mjasrotia) [inline components/mixins/experience-event/experienceevent-claim-process.schema.json:41]: type ?

File: components/mixins/experience-event/experienceevent-contact-request-details.schema.json [hunk 12]
```diff
@@ -0,0 +1,105 @@
...
+    "manufacturing",
+    "media_and_entertainment",
+    "public_sector"
+  ],
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "description": "Visitor contact details such as chat start and end, as well as click to email, text, and call counts. ",
+  "definitions": {
+    "contact-request-details": {
+      "type": "object",
+      "properties": {
```
Comment (mjasrotia) [inline components/mixins/experience-event/experienceevent-contact-request-details.schema.json:32]: type and properties not required as its a definition

File: components/mixins/experience-event/experienceevent-contact-request-details.schema.json [hunk 13]
```diff
@@ -0,0 +1,105 @@
...
+        "xdm:customerFeedback": {
+          "type": "object",
+          "properties": {
+            "xdm:contactRequestDetails": {
+              "type": "object",
```
Comment (mjasrotia) [inline components/mixins/experience-event/experienceevent-contact-request-details.schema.json:37]: missing properties

File: components/mixins/experience-event/experienceevent-dispute-steps.schema.json [hunk 14]
```diff
@@ -0,0 +1,38 @@
...
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:industry": ["financial_services"],
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "description": "Dispute steps such as start, save, failure, and complete.",
+  "definitions": {
+    "dispute-steps": {
+      "type": "object",
+      "properties": {
```
Comment (mjasrotia) [inline components/mixins/experience-event/experienceevent-dispute-steps.schema.json:20]: type and properties not required

File: components/mixins/experience-event/experienceevent-dispute-steps.schema.json [hunk 15]
```diff
@@ -0,0 +1,38 @@
...
+        "xdm:disputes": {
+          "type": "object",
+          "properties": {
+            "xdm:disputeSteps": {
+              "type": "object",
```
Comment (mjasrotia) [inline components/mixins/experience-event/experienceevent-dispute-steps.schema.json:25]: properties missing

File: components/mixins/experience-event/experienceevent-contact-request-details.schema.json [hunk 16]
```diff
@@ -0,0 +1,106 @@
...
+                  "title": "Chat End",
+                  "$ref": "https://ns.adobe.com/xdm/datatypes/data/measure",
+                  "description": "Captured when a chat is ended with a visitor."
+                },
+                "xdm:chatOffered": {
+                  "title": "Chat Offered",
+                  "$ref": "https://ns.adobe.com/xdm/datatypes/data/measure",
+                  "description": "Captures the number of times a chat session has been offered to a visitor."
+                },
+                "xdm:chatSessionId": {
```
Comment (mjasrotia) [inline components/mixins/experience-event/experienceevent-contact-request-details.schema.json:53]: ID

File: components/mixins/experience-event/experienceevent-credit-limit-increase-details.schema.json [hunk 17]
```diff
@@ -0,0 +1,38 @@
...
+  "title": "Credit Limit Increase Details",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:industry": ["financial_services"],
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "description": "Credit limit increase application details such as ID, type, approval, and status.",
+  "definitions": {
+    "credit-limit-increase-details": {
+      "type": "object",
```
Comment (mjasrotia) [inline components/mixins/experience-event/experienceevent-credit-limit-increase-details.schema.json:19]: type not required in the definition

File: components/mixins/experience-event/experienceevent-credit-limit-increase-details.schema.json [hunk 18]
```diff
@@ -0,0 +1,38 @@
...
+      "properties": {
+        "xdm:personalFinances": {
+          "type": "object",
+          "properties": {
+            "xdm:creditLimitIncreaseDetails": {
+              "type": "object",
```
Comment (mjasrotia) [inline components/mixins/experience-event/experienceevent-credit-limit-increase-details.schema.json:25]: properties missing

File: components/mixins/experience-event/experienceevent-deposit-details.schema.json [hunk 19]
```diff
@@ -0,0 +1,43 @@
...
+  "title": "Deposit Details",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:industry": ["financial_services"],
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "description": "Deposit details such as ID, type, and amount.",
+  "definitions": {
+    "deposit-details": {
+      "type": "object",
```
Comment (mjasrotia) [inline components/mixins/experience-event/experienceevent-deposit-details.schema.json:19]: type not required in definitions

File: components/mixins/experience-event/experienceevent-deposit-details.schema.json [hunk 20]
```diff
@@ -0,0 +1,43 @@
...
+      "properties": {
+        "xdm:personalFinances": {
+          "type": "object",
+          "properties": {
+            "xdm:depositDetails": {
+              "type": "object",
+              "xdm:deposit": {
```
Comment (mjasrotia) [inline components/mixins/experience-event/experienceevent-deposit-details.schema.json:26]: properties missing

File: components/datatypes/financial-account.schema.json [hunk 21]
```diff
@@ -33,15 +33,14 @@
...
-            "$ref": "https://ns.adobe.com/experience/campaign/experienceevent-profile-personal-details"
-          }
+            "$ref": "https://ns.adobe.com/xdm/context/profile-personal-details"
+          },
+          "description": "Array of the owner(s) of the selected financial account. Extension of the personal profile details data type."
         },
         "xdm:financialAccountType": {
           "title": "Financial Account Type",
           "enum": ["checking", "savings", "retirement", "investment"],
-          "type": "string",
```
Comment (mjasrotia) [inline components/datatypes/financial-account.schema.json:44]: Type missing

File: components/datatypes/policy-details.schema.json [hunk 22]
```diff
@@ -0,0 +1,59 @@
...
```
Comment (mjasrotia) [inline components/datatypes/policy-details.schema.json:46]: type  still missing

File: components/mixins/experience-event/experienceevent-claim-process.schema.json [hunk 23]
```diff
@@ -0,0 +1,75 @@
...
+                  "title": "Claim Steps",
+                  "$ref": "https://ns.adobe.com/xdm/datatypes/tool-usage",
+                  "description": "Extension of toolUsage data type."
+                },
+                "xdm:claimType": {
+                  "title": "Claim Type",
+                  "type": "string",
+                  "description": "Type of claim submitted."
+                },
+                "xdm:policy": {
```
Comment (mjasrotia) [inline components/mixins/experience-event/experienceevent-claim-process.schema.json:57]: Should we name this as claimPolicy as "policy" is too generic to be under "xdm" namespace.

File: components/mixins/experience-event/experienceevent-contact-request-details.schema.json (hunk 16 above)
Comment (mjasrotia) [reply components/mixins/experience-event/experienceevent-contact-request-details.schema.json:53]: Please change "id" to "ID"

**Review by mjasrotia — APPROVED** [review]

**Review by prabhum2 — APPROVED** [review]

**Pull request merged**